omod/src/main/resources/config.xml

```diff
@@ -86,37 +90,38 @@
 		<property>${project.parent.artifactId}.openempi.globalIdentifierDomainId</property>
 		<defaultValue></defaultValue>
 		<description>
-			Specifies main identifier on MPI server (e.g. "xyz" for OpenEMPI)
+			Specifies main identifier on MPI server (e.g. "xyz" for OpenEMPI) (GSOC OpenEMPI Specific Implementation)
 		</description>
 	</globalProperty>
 
 	<globalProperty>
 		<property>${project.parent.artifactId}.mpi.implementation</property>
-		<defaultValue></defaultValue>
+		<defaultValue>registrationcore.mpi.implementation.PixPdq</defaultValue>
 		<description>
-			Which MPI to should we connect to? Specify a Spring bean name, or leave blank to disable MPI integration
+			Which MPI to should we connect to? Specify a Spring bean name, or leave blank to disable MPI integration. Options include:
+			"registrationcore.mpi.implementation.PixPdq", "registrationcore.mpi.implementation.OpenEMPI".
 		</description>
 	</globalProperty>
 
 	<globalProperty>
 		<property>${project.parent.artifactId}.mpi.url</property>
 		<defaultValue></defaultValue>
 		<description>
-			Specifies url to MPI server
+			Specifies url to MPI server. (GSOC OpenEMPI Specific Implementation)
 		</description>
 	</globalProperty>
 
 	<globalProperty>
 		<property>${project.parent.artifactId}.mpi.username</property>
-		<defaultValue></defaultValue>
+		<defaultValue>admin</defaultValue>
 		<description>
 			Specifies username for authentication to MPI server
 		</description>
 	</globalProperty>
 
 	<globalProperty>
-		<property>${project.parent.artifactId}.mpi.username</property>
-		<defaultValue></defaultValue>
+		<property>${project.parent.artifactId}.mpi.password</property>
+		<defaultValue>admin</defaultValue>
```

Comment:
We generally do not recommend storing passwords in global properties. Did you evaluate running it instead as a daemon user account?

Reply:
I did not, I think what I'll do is leave it as this for now but then create a ticket and do a separate pull request after this is merged. Does that sound good? I'm not sure what running it as a daemon user account involves so some guidance would be helpful!

Reply:
@dkayiwa this is not a password for an OpenMRS account, it's a password to the MPI so daemon user doesn't apply. But I still agree that we should always avoid storing passwords as GP since they get displayed as text in the UI, I don't know what the solution is but for now I'd ignore this. I personally think we should possibly consider using environment variables but this is a design forum topic.

Reply:
Moved discussion to talk